services/catarse/app/lib/business_calendar.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class BusinessCalendar
```

Comment:
**Sugestão**
Ao invés de criar uma classe, cria um initializer e lá tu inicializa uma variável global:
```
BusinessCalendar = Business::Calendar.new(...)
```
Assim o objeto será carregado apenas uma vez, e quem for utilizar não precisará criar uma instância nova toda vida.